rules/prefer-map-has.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const MESSAGE_ID_ERROR = 'prefer-map-has/error';
+const MESSAGE_ID_SUGGESTION = 'prefer-map-has/suggestion';
+const messages = {
+	[MESSAGE_ID_ERROR]: 'Prefer `{{replacement}}` over `{{value}}`.',
+	[MESSAGE_ID_SUGGESTION]: 'Replace `{{value}}` with `{{replacement}}`.',
+};
+
+/**
+ * @param {import('eslint').Rule.RuleContext} context
+ * @param {import('estree').SimpleCallExpression} node
+ * @param {import('estree').Node} [replacement]
+ */
```

Comment:
```suggestion
/**
@param {import('eslint').Rule.RuleContext} context
@param {import('estree').SimpleCallExpression} node
@param {import('estree').Node} [replacement]
*/
```

Reply:
This doesn't seem to be the recommended style for `jsdoc`

Reply:
It's the style we use.